scripts/init.js

```diff
@@ -29,6 +29,11 @@ module.exports = function(hostPath, appName, verbose) {
     hostPackage.scripts[command] = 'react-scripts ' + command;
   });
 
+  // explicitly specify ESLint config path for editor plugins
+  hostPackage.eslintConfig = {
+    extends: "./node_modules/react-scripts/config/eslint.js",
```

Comment:
./node_modules/ should be unnecessary, I think. Node resolution means you should be able to start right with react-scripts.